src/actor/Claimer.spec.ts

```diff
@@ -395,5 +449,86 @@ describe('Claimer', () => {
         )
       )
     })
+    it('Should throw, when claimer tries to make a request for attestation, without a PE-enabled identity', async () => {
+      blockchainApi.query.attestation.attestations.mockReturnValue(
+        mockChainQueryReturn('attestation', 'attestations', [
+          cType.hash,
+          attester.getAddress(),
+          undefined,
+          0,
+        ])
+      )
+
+      const claimerWithoutPE = await Identity.buildFromURI('//bob', {
+        peEnabled: false,
+      })
+
+      const { message: initAttestation } = await Attester.initiateAttestation(
+        attester,
+        claimerWithoutPE.getPublicIdentity()
+      )
+
+      await expect(
+        Claimer.requestAttestation(
+          claim,
+          claimerWithoutPE,
+          attester.getPublicIdentity(),
+          {
+            initiateAttestationMsg: initAttestation,
+          }
+        )
+      ).rejects.toThrowError(ERROR_IDENTITY_NOT_PE_ENABLED())
+    })
+
+    it('Should throw, when claimer tries to make a credential, without a PE-enabled identity', async () => {
+      blockchainApi.query.attestation.attestations.mockReturnValue(
+        mockChainQueryReturn('attestation', 'attestations', [
+          cType.hash,
+          attester.getAddress(),
+          undefined,
+          0,
+        ])
+      )
+
+      const claimerWithoutPE = await Identity.buildFromURI('//bob', {
```

Comment:
See above.

```suggestion
      const claimerWithoutPE = await Identity.buildFromURI('//Bob', {
```

Reply:
fixed in ea38166a4ce1b9c442db0264098e7f7ff080847f